stock_inventory_line_price/views/inventory_view.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <record id="view_inventory_form" model="ir.ui.view">
+        <field name="name">view.inventory.form.inherit</field>
+        <field name="model">stock.inventory</field>
+        <field name="inherit_id" ref="stock.view_inventory_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='line_ids']//field[@name='product_id']" position="after">
+                <field name="theoretical_std_price"/>
+                <field name="standard_price"/>
+            </xpath>
+            <xpath expr="//tree" position="attributes">
+                <attribute name="colors">blue: product_qty != theoretical_qty or theoretical_std_price != standard_price; red: theoretical_qty &lt; 0</attribute>
```

Comment:
We are still having the same issue here as in [v9](https://github.com/OCA/stock-logistics-warehouse/pull/275#pullrequestreview-37237048)